db/seeds.rb

```diff
@@ -38,15 +39,29 @@
 puts "Creating kondos"
 
 20.times do
-    kondos =  Kondo.new(
-                name: ["Interior Designer","House Keeping", "Home Destroyer", "Thief"].sample,
-                summary: Faker::Lorem.sentence(word_count: 20, supplemental: true, random_words_to_add: 10),
-                details: Faker::Lorem.sentence(word_count: 50, supplemental: true, random_words_to_add: 50),
-                prefecture: ["Tokyo", "Kanagawa", "Chiba", "Saitama", "Ibaraki", "Tochigi", "Yamanashi"].sample,
-                price: ((rand(5..20)) * 1000),
-                user_id: [user1, user2, user3, user4].sample.id
-                )
-    kondos.save!
+  kondos = Kondo.new(
+              name: ["Interior Designer","House Keeping", "Home Destroyer", "Thief"].sample,
+              summary: Faker::Lorem.sentence(word_count: 20, supplemental: true, random_words_to_add: 10),
+              details: Faker::Lorem.sentence(word_count: 50, supplemental: true, random_words_to_add: 50),
+              prefecture: ["Tokyo", "Kanagawa", "Chiba", "Saitama", "Ibaraki", "Tochigi", "Yamanashi"].sample,
+              price: ((rand(5..20)) * 1000),
+              user_id: [user1, user2, user3, user4].sample.id
+              )
+  kondos.save!
 end
 
 puts "Kondos Creation done!"
+
+puts "Creating bookings"
+
+kondos = Kondo.all
+
+10.times do
+  bookings = Booking.new(
+    user_id: [user1, user2, user3, user4].sample.id,
+    kondo_id: kondos.sample.id,
+    confirmed: false,
+    booked_date: Time.now
+  )
+  bookings.save!
+end
```

Comment:
Minor, however please do add a puts to indicate that booking creation has been completed.

Reply:
Noted! It's in my second commit (I realized right before opening the pull request so I committed twice, sorry about the confusion!

Reply:
No worries sorry for missing it.